worldedit-sponge/build.gradle.kts

```
@@ -1,54 +1,87 @@
 import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+import org.spongepowered.gradle.plugin.config.PluginLoaders
+import org.spongepowered.plugin.metadata.model.PluginDependency
 
 plugins {
     id("org.spongepowered.gradle.plugin")
+    id("org.spongepowered.gradle.vanilla")
 }
 
 applyPlatformAndCoreConfiguration()
 applyShadowConfiguration()
 
-// I can't believe sponge sets this in a base plugin with no opt-out
-convention.getPlugin(JavaPluginConvention::class.java).apply {
-    setSourceCompatibility(null)
-    setTargetCompatibility(null)
+repositories {
+    mavenCentral()
 }
 
-repositories {
-    maven { url = uri("https://repo.codemc.org/repository/maven-public") }
+minecraft {
+    version("1.16.5")
+}
+
+val spongeApiVersion = "8.0.0";
+
+sponge {
+    apiVersion(spongeApiVersion)
+    license("GPL3")
```

```suggestion
    license("GPL-3.0-or-later")
```
it ain't officially spdx but that doesn't mean we can't do it